src/DI/ContainerBuilder.php

```diff
@@ -174,24 +168,21 @@ public function getAliases()
 
 
 	/**
+	 * @param  string[]
 	 * @return self
 	 */
-	public function setClassName($name)
```

Comment:
Most likely nobody was using this method, but just removing seems a bit aggresive. But the trigger is good https://github.com/nette/di/commit/e3c82aa62d87506e33c8a73c2a87fecadf6b76fb :+1: